site/using-weave/service-management.md

```diff
@@ -38,6 +39,17 @@ With the above in place, you are ready to connect to the 'nc' service from anywh
 
 Similar NAT rules to the above can used to expose services not just to the outside world but also other, internal, networks.
 
+###<a name="export-route"></a> Exporting Services to the wider network
+
+If you want hosts that are not running Weave Net to access services
+exposed as above, on each of those machines:
```

Comment:
Should also use title case for the heading, Exporting Services to the Wider Network  -- rule is to capitalize words with four or more letters.